src/expressions.rs

```diff
@@ -10,10 +10,13 @@ pub(crate) fn chat_display_name(chat_expression: &gtk::Expression) -> gtk::Expre
         let chat = args[1].get::<Chat>().unwrap();
         let title = args[2].get::<String>().unwrap();
         if chat.is_own_chat() {
-            gettext("Saved Messages")
-        } else {
-            title
+            return gettext("Saved Messages");
+        } else if let ChatType::Private(user) = chat.type_() {
+            if let UserType::Deleted = user.type_().0 {
```

Comment:
^^^ The same thing as the review above should be applied here. So I suggest creating a function containing the expression you need in this module, so that we can reuse it more easily.